erizo_controller/erizoClient/src/Room.js

```diff
@@ -720,7 +724,8 @@ Erizo.Room = function (spec) {
                 // 1- Subscribe to Stream
 
                 if (that.p2p) {
-                    sendSDPSocket('subscribe', {streamId: stream.getID()});
+                    sendSDPSocket('subscribe', {streamId: stream.getID(),
```

Comment:
nope, but it will have if we add logs to ErizoController in the short future (about an hour 😄 )